app/resources/en/translation/deceased/address.json

```diff
@@ -24,7 +24,7 @@
     },
     "postTown": {
       "required": {
-        "summary": "Enter a town or city",
+        "summary": "You haven&rsquo;t entered the town or city",
```

Comment:
Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence